README.md

```diff
@@ -30,6 +30,11 @@ The following command line utilities are included in the PROJ package:
 
 
 ## Installation
+### Build dependencies
+
+PROJ requires a C and C++ compiler that supports the C++11 standard.
```

Comment:
or shorter "PROJ requires a C and C++11 compiler", also makes it clear C++11 standard requirement does not apply to the C compiler :)